site/pages/sysadmin/org.page.tsx

```diff
@@ -160,12 +161,16 @@ export const getServerSideProps = async (
         fetchAdminAreaCodes(),
     ]);
 
-    const pageState = getPageState<Organisation>(
-        errorCookie,
-        organisationSchema,
-        "ID",
-        !errorCookie && orgId && orgInfo ? orgInfo : undefined,
-    );
+    let pageState: PageState<Partial<Organisation>> = {
+        errors: [],
+        inputs: {},
+    };
+
+    if (errorCookie) {
+        pageState = JSON.parse(errorCookie) as PageState<Partial<Organisation>>;
```

Comment:
could we not use zod to parse this?

Reply:
As the errorCookie is json string we can use zod to parse it